eos/b-decays/b-to-dstar-l-nu.cc

```diff
@@ -93,146 +269,272 @@ namespace eos
-            // note that upper s limit is now less than the B->Dlnu case
-            br_taus = integrate<GSL::QAGS>(f, s_min_tau, s_max);
+            
+            br_tau = this->normalized_integrated_branching_ratio(s_min_tau, s_max_tau);
         }
-        return br_taus / br_muons;
+
+        return br_tau / br_mu;
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_amplitude_polarization_L(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.normalized_amplitude_polarization_L() * std::norm(_imp->model->ckm_cb());
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_amplitude_polarization_T(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.normalized_amplitude_polarization_T() * std::norm(_imp->model->ckm_cb());
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_f_L(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.f_L();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_c_1(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_c_1();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_c_2(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_c_2();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_c_3(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_c_3();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_t_1(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_t_1();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_t_2(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_t_2();
+    }
+
+    double
+    BToDstarLeptonNeutrino::integrated_a_t_3(const double & s_min, const double & s_max) const
+    {
+        auto   o = _imp->integrated_angular_observables(s_min, s_max);
+        return o.a_t_3();
+    }
+
+    //* cf. [DSD2014], eq. (6), p. 5
+    double
+    BToDstarLeptonNeutrino::four_differential_decay_width(const double & s, const double & c_theta_l, const double & c_theta_d, const double & phi) const
+    {
+        // compute d^4 Gamma, cf. [DSD2014], p. 5, eq. (6)
+        // Trigonometric identities: Cosine squared of the angles
+        double c_theta_d_2 = c_theta_d * c_theta_d;
+        double c_theta_l_2 = c_theta_l * c_theta_l;
+        double c_phi = cos(phi);
```

Comment:
Can you use here 

  "cos" -> "std::cos" 
  "sin" -> "std::sin"
  "sqrt" -> "std::sqrt"

unless you have written somewhere

  using std::cos, std::sin, std::sqrt;

Reply:
done!